src/pokedex_plus_hgss.c

```diff
@@ -4088,6 +4088,12 @@ static void UNUSED HighlightScreenSelectBarItem(u8 selectedScreen, u16 unused)
 #define tPersonalityLo data[14]
 #define tPersonalityHi data[15]
 
+// Types palettes need to be loaded at a different slot than anticipated by gTypesInfo
+// to avoid overlapping with caught mon sprite platte slot
+// Normarl type info palette slots: 13 , 14 and 15
+// Caugh mon palette slot: 15
```

Comment:
```suggestion
// to avoid overlapping with caught mon sprite palatte slot
// Normal type info palette slots: 13 ,14 and 15
// Caught mon palette slot: 15
```